### PR TITLE
[5.2] Add active class to active item in mod_ articles

### PR DESCRIPTION
**Pull request by RickR2H**
Pull Request for Issue # .

### Summary of Changes
The new mod_articles module has no active class for the active item like there was in mod_articles_category. This PR adds this functionality. 

### Testing Instructions
1. Create a list of articles with the module active on all pages.
2. Open the inspector in the browser.
3. With the Title Only (lists) option selected, see if the link to the current article has an active class.
image

### Actual result BEFORE applying this Pull Request
No active class

### Expected result AFTER applying this Pull Request
Has an active class

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [ ] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [ ] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by brianteeman** [comment]
There is an active class already. Maybe its in the wrong place? I doubt we need two
https://github.com/joomla/joomla-cms/blob/20cfbbf13976575ce68c84cc150e0aed5ff4389b/modules/mod_articles/tmpl/default_items.php#L37

image

**Comment by RickR2H** [comment]
Agreed that with :has selector the article element now can be selected. Is a coverage of 91,66% acceptable to remove the active class on the article element?

File: modules/mod_articles/tmpl/default_titles.php [hunk 1]
```diff
@@ -14,7 +14,7 @@
 <ul class="mod-articles mod-list">
     <?php foreach ($items as $item) : ?>
         <li itemscope itemtype="https://schema.org/Article">
-            <a href="<?php echo $item->link; ?>" itemprop="url">
+            <a <?php echo $item->active ? ' class="' . $item->active . '"' : ''; ?>href="<?php echo $item->link; ?>" itemprop="url">
```
Comment (Quy) [inline modules/mod_articles/tmpl/default_titles.php]:
```suggestion
            <a <?php echo $item->active ? 'class="' . $item->active . '" ' : ''; ?>href="<?php echo $item->link; ?>" itemprop="url">
```

Comment (RickR2H) [reply modules/mod_articles/tmpl/default_titles.php]: Nice catch! Thanks.

**Comment by RickR2H** [comment]
@brianteeman please test the active link for the article list.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on b2494eb188bceb48a419ac848d7c94fa9fd02230<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44153">issues.joomla.org/tracker/joomla-cms/44153</a>.</sub>

**Comment by BertaOctech** [comment]
I have tested this item :white_check_mark: successfully on b2494eb188bceb48a419ac848d7c94fa9fd02230<br /><br />I have followed test instructions.
Active class appeared in title when the patch was applied, even with Title Only.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44153">issues.joomla.org/tracker/joomla-cms/44153</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44153">issues.joomla.org/tracker/joomla-cms/44153</a>.</sub>

**Pull request merged**